cli/src/cli.ts

```diff
@@ -33,13 +34,13 @@ const program = yargs
     .option('header', {
         alias: 'H',
         type: 'array',
-        // string: true,
```

Comment:
don't know why this was commented out. from what I see, validating scalars and headers as string makes sense.